README.md

```diff
@@ -1,3 +1,224 @@
+
+```sh
+fuelup show
+```
+
+The command above should yield a similar output below
+
+```sh
+active toolchain
+----------------
+latest-aarch64-apple-darwin (default)
+  forc : 0.63.3
+    - forc-client
+      - forc-deploy : 0.63.3
+      - forc-run : 0.63.3
+    - forc-crypto : 0.63.3
+    - forc-debug : 0.63.3
+    - forc-doc : 0.63.3
+    - forc-fmt : 0.63.3
+    - forc-lsp : 0.63.3
+    - forc-tx : 0.63.3
+    - forc-wallet : 0.9.0
+  fuel-core : 0.34.0
+  fuel-core-keygen : 0.34.0
+
+fuels versions
+--------------
+forc : 0.66.1
+forc-wallet : 0.66.0
+```
+
+## Proxy
+
+> DO NOT ATTEMPT TO USE THE SRC-14 STANDARD OR UPGRADABILITY DIRECTLY YOURSELF TO AVOID COLLISION CATASTROPHES. THEY ARE FOR EDUCATIONAL PURPOSES ONLY. 
+
+> Please refer to [using the Proxy](#using-the-proxy) section to safely incoporate proxies into your project.
+
+
+#### [SRC-14 Simple Upgradeable Proxies](https://docs.fuel.network/docs/sway-standards/src-14-simple-upgradeable-proxies/)
+
+The SRC-14 standard is similar to upgradeable proxies in Ethereum like UUPS, where an implementation/target contract is stored in storage of the proxy contract and all calls are delegated to it. 
+
+In FuelVM, this behavior is implemented using the `LDC` instruction, which Sway's `std::execution::run_external` leverages to mimic EVM's `delegatecall`, allowing execution of instructions from another contract while retaining the original contract's storage context.
+
+Essential functions to note are `set_proxy_target(new_target: ContractId)` and `proxy_target()` as mentioned above.
+
+```sway
+abi SRC14 {
+    #[storage(read, write)]
+    fn set_proxy_target(new_target: ContractId);
+    #[storage(read)]
+    fn proxy_target() -> Option<ContractId>;
+}
+ 
+abi SRC14Extension {
+    #[storage(read)]
+    fn proxy_owner() -> State;
+}
+```
+
+#### [Upgradable Libraries](https://docs.fuel.network/docs/sway-libs/upgradability/)
+
+To use the SRC-14 standard along with the upgradability library, you simply need to import the necessary modules as shown below:
+
+```sway
+use sway_libs::upgradability::*;
+use standards::{src14::*, src5::*};
+```
+
+With these imports, your implementation file will look like this
+
+```sway
+use sway_libs::upgradability::{_proxy_owner, _proxy_target, _set_proxy_target};
+use standards::{src14::{SRC14, SRC14Extension}, src5::State};
+ 
+#[namespace(SRC14)]
+storage {
+    // target is at sha256("storage_SRC14_0")
+    target: Option<ContractId> = None,
+    proxy_owner: State = State::Uninitialized,
+}
+ 
+impl SRC14 for Contract {
+    #[storage(read, write)]
+    fn set_proxy_target(new_target: ContractId) {
+        _set_proxy_target(new_target);
+    }
+ 
+    #[storage(read)]
+    fn proxy_target() -> Option<ContractId> {
+        _proxy_target()
+    }
+}
+ 
+impl SRC14Extension for Contract {
+    #[storage(read)]
+    fn proxy_owner() -> State {
+        _proxy_owner(storage.proxy_owner)
+    }
+}
+```
+
```

Comment:
Can we add a section about the [owned proxy implementation from sway-standard-implementations](https://github.com/FuelLabs/sway-standard-implementations/tree/master/src14/owned_proxy) here.
 
along the lines of:
> The [Owned Proxy Contract](https://github.com/FuelLabs/sway-standard-implementations/blob/master/src14/owned_proxy/contract/src/main.sw) is an opinionated implementation of the [extended SRC-14 standard](https://docs.fuel.network/docs/sway-standards/src-14-simple-upgradeable-proxies/). It utilises the [Upgradability library from sway-libs](https://github.com/FuelLabs/sway-libs) and includes initialization functionality that allows for secure ownership upon deployment.

and we should specify that it is this specific contract that is pre-compiled and used in the forc deploy proxy feature.